orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/monitoreddeploy/BroadcastDeployStartingStage.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.pipeline.monitoreddeploy;
+
+import com.netflix.spinnaker.orca.clouddriver.tasks.monitoreddeploy.BroadcastDeployStartingTask;
+import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder;
+import com.netflix.spinnaker.orca.pipeline.TaskNode;
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConditionalOnProperty(value = "monitored-deploy.enabled")
+public class BroadcastDeployStartingStage implements StageDefinitionBuilder {
+  @Override
+  public void taskGraph(Stage stage, TaskNode.Builder builder) {
+    builder.withTask("broadcastDeployStarting", BroadcastDeployStartingTask.class);
```

Comment:
Is this truly "starting" or has the deploy already started? I'm quite reluctant to broadcast events in the present tense. To me, present tense implies listeners are synchronous interceptors that can change the behavior of a deploy (e.g. mutate the deploy request, or even cancel it before it has a chance to start).